test/ctia/http/routes/pagination_test.clj

```diff
@@ -0,0 +1,142 @@
+(ns ctia.http.routes.pagination-test
+  (:refer-clojure :exclude [get])
+  (:require
+   [clojure.test :refer [deftest is testing use-fixtures join-fixtures]]
+   [schema-generators.generators :as g]
+   [ctia.test-helpers.core :refer [delete get post put] :as helpers]
+   [ctia.test-helpers.fake-whoami-service :as whoami-helpers]
+   [ctia.test-helpers.store :refer [deftest-for-each-store]]
+   [ctia.test-helpers.auth :refer [all-capabilities]]
+   [ctia.schemas.judgement :refer [NewJudgement]]))
+
+(use-fixtures :once (join-fixtures [helpers/fixture-schema-validation
+                                    helpers/fixture-properties:clean
+                                    whoami-helpers/fixture-server]))
+
+(use-fixtures :each whoami-helpers/fixture-reset-state)
+
+(deftest-for-each-store test-observable-routes-paging
+  (helpers/set-capabilities! "foouser" "user" all-capabilities)
+  (whoami-helpers/set-whoami-response "45c1f5e3f05d0" "foouser" "user")
+
+  (testing "with 30 judgements test setup"
+    (let [doc-count 30
+          headers {"api_key" "45c1f5e3f05d0"}
+          new-judgements (->> (g/sample doc-count NewJudgement)
+                              (map #(merge % {:observable {:type "ip"
+                                                           :value "1.2.3.4"}
+                                              :disposition 5
+                                              :disposition_name "Unknown"})))
+          judgement-post-responses (map #(post "ctia/judgement"
+                                               :body %
+                                               :headers headers) new-judgements)
+          stored-judgements (->> judgement-post-responses
+                                 (map :body))]
+
+      (is (= doc-count (count stored-judgements)))))
```

Comment:
Wouldn't it be better to add a test for each entity, just to verify, we didn't forgot to add pagination for some routes?